### DISCUSSION TITLE
Where to find a mapping from the metrics names in docs to metrics names in Prometheus

### DISCUSSION THREAD

**Discussion by deil87** [milvus-io/milvus#4468] [Q&A and General discussion] [answered]
From the docs here https://milvus.io/docs/v0.11.0/milvus_metrics.md#Milvus-performance-metrics we know the names of the metrics but how can we figure out metrics' names that are used in Prometheus/Grafana?  

On a provided dashboard https://milvus.io/docs/v0.11.0/setup_grafana.md#Configure-and-start-Grafana we don't see let's say `Query Time per Vector`.

**Comment by jingkl** [comment, marked as answer]
![1608520427(1)](https://user-images.githubusercontent.com/34296482/102736131-283e2100-437f-11eb-8384-91529f07c45d.png)
We can see Milvus metrics in Grafana Dashboard for Milvus. 0.11 is currently built and some metrics cannot be displayed, but this issue will be fixed in 0.11.1
